aiohttp/log.py

```diff
@@ -1,7 +1,6 @@
 import logging
 
 
-access_logger = logging.getLogger('aiohttp.access')
```

Comment:
Strictly speaking all names not enumerated in `aiohttp.__all__` and `aiohttp.web.__all__` are private.
Loggers should be accessed by names, e.g. `getLogger('aiohttp.server')`